### ISSUE TITLE
Feature request:  select and operate on multiple folders

### ISSUE THREAD

**Issue by wealthychef1** [ozntel/file-tree-alternative#106]
There appears to be no way to select multiple folders that I can tell.  I tried shift-select, which is pretty universal across platforms, but this does not work in the FTA plugin's interface.   For that matter, it does not work in Obsidian's file browser either, which I find very curious. 

Would it be possible to provide this?  I'd like to be able to move multiple folders in my vault from within Obsidian and so far I have not found this.

**Comment by dawnlh** [comment]
Meet the same problem. Some other similar limitations include cannot use Ctrl+C and Ctrl+V for file copy and paste, cannot drag files and move them to other folders. It's very inconvenient.

**Comment by wealthychef1** [comment]
> Meet the same problem. Some other similar limitations include cannot use Ctrl+C and Ctrl+V for file copy and paste, cannot drag files and move them to other folders. It's very inconvenient.

Have you tried option-clicking on the folders?  In Obsidian, this is the only way to select multiple folders.  But it works for me on MacOS.  I assume it works the same on Windows.

**Comment by dawnlh** [comment]
I tried Ctrl+clicking on Win10, But it dnd't work for me.  However, shift + clicking can be used to select consecutive files.

**Comment by ksdavidc** [comment]
I second this. It is definitely something that is missing, especially for moving multiple files from one folder to another. In the regular file explorer, this is difficult, and it would be so easy on file tree, IF...

**Comment by anatoliibakhtin** [comment]
yes, the function of working with multiple files is very lacking, as it is possible in the standard file tree 
![image](https://github.com/ozntel/file-tree-alternative/assets/71871141/3fefc89a-335d-475e-ab84-2de42f226cd4)

**Comment by ksdavidc** [comment]
BUMP!

**Comment by wealthychef1** [comment]
dead
